docs/images/how_to_use_logo_template.md

```diff
@@ -0,0 +1,6 @@
+# How to use the logo template
+
+1. Replace `pipelinename` with the actual name of the pipeline
+2. Save the slide as a `sanger-tol-<pipeline>_logo.png`
```

Comment:
```suggestion
2. _Export_ the slide as a `sanger-tol-<pipeline>_logo.png`
```

Reply:
Added the minimum resolution

Reply:
How do you export the slide ? When I click "Export: and select PNG, I'm asked for a resolution in pixels, not pixels/inch
![Screenshot 2022-07-20 at 11 54 35](https://user-images.githubusercontent.com/623458/179965610-95c27644-2e47-4b7b-9b76-9c4e451f7a66.png)

Reply:
I have made the changes to include those descriptors as well. I generally do a default export and then use tools in preview to make sure I have the right resolution.